edgelet/iotedged/src/lib.rs

```diff
@@ -253,7 +253,7 @@ where
 
         if let Provisioning::Manual(ref manual) = settings.provisioning() {
             if let ManualAuthMethod::DeviceConnectionString(cs) = manual.authentication_method() {
-                if cs.device_connection_string() == DEFAULT_CONNECTION_STRING {
+                if cs.device_connection_string().is_empty() {
```

Comment:
The call to `manual_provision` below will already fail with `ErrorKind::ConnectionStringEmpty` as part of calling `ManualDeviceConnectionString::parse_device_connection_string`, so this extra check isn't necessary.